addon/record-data.js

```diff
@@ -348,158 +475,366 @@ export default class FragmentRecordData extends RecordData {
       @private
     */
   changedAttributes() {
```

Comment:
`changedAttributes` previously returned `[ currentFragment, currentFragment ]` where `currentFragment` is a Fragment instance.

Now it returns `[ oldState, currentState ]` where `oldState` is the fragment's canonical state serialized and `currentState` is the fragment's dirty state serialized.

Reply:
as it should <3